tests/api_test.py

```diff
@@ -121,6 +122,18 @@ def test_create_by_wrong_scs(self):
         assert len(results) == 1
         assert results[0] is False
 
+    def test_create_by_scs_with_output_struct(self):
+        self.get_server_message('{"errors": [], "id": 1, "event": false, "status": true, "payload": [1]}')
+        results = client.create_elements_by_scs([SCs("concept1 -> node1;;", ScAddr(0))])
+        assert len(results) == 1
+        assert results[0] is True
+
+    def test_create_by_wrong_scs_with_output_struct(self):
+        self.get_server_message('{"errors": [], "id": 1, "event": false, "status": true, "payload": [0]}')
+        results = client.create_elements_by_scs([SCs("concept1 -> ", ScAddr(0))])
+        assert len(results) == 1
+        assert results[0] is False
```

Comment:
message with error description is not returned if error was returned from ScServer? Is it ok? 
There is no tests for passing multiple scs-lines, but format of the request looks like we can do it. Can you add tests for creating multiple scs with multiple output structs> What happend if I will pass a multiple SCs with a common structure?

Reply:
Add tests for multiple SCs